src/GraphQL/Conversion/ValueConverter.cs

```diff
@@ -132,13 +132,19 @@ static ValueConverter()
             Register<char, int>(value => Convert.ToInt32(value));
         }
 
+        /// <summary>
+        /// Returns an object of the specified type and whose value is equivalent to the specified object.
+        /// </summary>
         public static T ConvertTo<T>(object value)
         {
             object v = ConvertTo(value, typeof(T));
 
             return v == null ? default : (T)v;
         }
 
+        /// <summary>
+        /// Returns an object of the specified type and whose value is equivalent to the specified object.
```

Comment:
Consider to add `  /// <exception cref="InvalidOperationException">blabla</exception>` or extend summary to cover this case. This applies to all methods.

Reply:
While I agree with you, my goal within #1857 is to add `<summary>` tags to each member, plus enough notes that users can understand the purpose of each method where it isn't obvious.  I think this is achievable, over time, and important.  If I spend time adding tags for parameters, exception, and return values to each member, the work will never get done.  Most parameters, exceptions, and return values are obvious.

Note that in this particular case, `InvalidOperationException` only occurs when there is no conversion registered; but we do not know what exceptions would be thrown if the conversion fails.  I'll add some notes here to the summary tag as it could be rather important when using these particular methods.

Reply:
> `InvalidOperationException` occurs when there is no conversion registered.

This will be quite enough.